OWNERS

```diff
@@ -3,14 +3,14 @@
 approvers:
  - dhellmann
  - hardys
+ - kashifest
+ - maelk
  - mhrivnak
  - russellb
- - maelk
- - kashifest
-
+ 
 reviewers:
- - Xenwar
- - Jaakko-Os
  - fmuyassarov
  - furkatgofurov7
+ - Jaakko-Os
```

Comment:
Should we remove Jaakko from here?

Reply:
Was not sure, if others also agree, I can remove.

Reply:
Since we have him in other repos also, let's keep for now, and remove at once from all repos if everyone agrees.